### PR TITLE
optimize: fix StyleCheck errors for hippo4j-agent-bootstrap

### LINKED ISSUE #1229: [TASK] About CheckStyle Errors
There are errors related to CheckStyle. I hope someone can fix them and make pull requests.

  - [ ] hippo4j-threadpool-infra-common @yhc777 
  - [x] hippo4j-threadpool-core @dmego 
  - [x] hippo4j-threadpool-adapter-web
  - [x] hippo4j-threadpool-spring-boot-starter-adapter-web
  - [x] hippo4j-threadpool-config-spring-boot-starter
  - [x] spring-plugin-common
  - [x] hippo4j-threadpool-config-spring-boot-1x-starter
  - [x] spring-boot-1.x-plugin
  - [x] spring-boot-2.x-plugin
  - [x] thread-pool-plugin
  - [x] apollo-plugin
  - [x] hippo4j-agent-bootstrap
  - [x] hippo4j-example-core
  - [x] hippo4j-threadpool-config-nacos-example
  - [x] hippo4j-threadpool-config-apollo-example
  - [x] hippo4j-threadpool-config-zookeeper
  - [x] hippo4j-threadpool-config-consul-example
  - [x] hippo4j-threadpool-config-etcd-example
  - [x] hippo4j-threadpool-config-nacos-spring-boot-1x
  - [x] hippo4j-threadpool-config-apollo-spring-boot-1x-example
  - [x] hippo4j-threadpool-server-spring-boot-starter
  - [x] hippo4j-threadpool-adapter-rabbitmq-example
  - [x] hippo4j-threadpool-adapter-stream-rabbitmq-example
  - [x] hippo4j-threadpool-adapter-stream-rocketmq-example
  - [x] hippo4j-threadpool-adapter-rocketmq-example
  - [x] hippo4j-threadpool-adapter-kafka-example
  - [x] hippo4j-threadpool-server-example
  - [x] hippo4j-threadpool-monitor-elasticsearch-example
  - [x] hippo4j-threadpool-rpc
  - [x] hippo4j-threadpool-server-config
  - [ ] hippo4j-agent-core

If everyone wants to fix those errors, they can change <skip.checkstyle.check>true</skip.checkstyle.check> to false and run mvn package to see more details.

Any pull request related to CheckStyle errors can be associated with this issue, and I will update this document when the PR is merged or find new errors.

### PR DESCRIPTION
**Pull request by DDreame**
Fixes #1229 

Changes proposed in this pull request:
- hippo4j-agent-bootstrap

> Check mailbox configuration when submitting. [Contributor Guide](https://hippo4j.cn/community/contributor-guide)

### CODE REVIEW

File: agents/threadpool/hippo4j-agent-bootstrap/src/main/java/cn/hippo4j/agent/bootstrap/Hippo4jAgent.java [hunk 1]
```diff
@@ -49,18 +48,17 @@
...
 import static net.bytebuddy.matcher.ElementMatchers.nameStartsWith;
 import static net.bytebuddy.matcher.ElementMatchers.not;
 
+/**
+ * hippo for java Agent
+ */
 public class Hippo4jAgent {
 
-    private static ILog LOGGER = LogManager.getLogger(Hippo4jAgent.class);
+    private static ILog logger = LogManager.getLogger(Hippo4jAgent.class);
```
Comment (magestacks) [inline agents/threadpool/hippo4j-agent-bootstrap/src/main/java/cn/hippo4j/agent/bootstrap/Hippo4jAgent.java]: Can it be capitalized if it is defined as final?

File: agents/threadpool/hippo4j-agent-bootstrap/src/main/java/cn/hippo4j/agent/bootstrap/Hippo4jAgent.java [hunk 2]
```diff
@@ -49,18 +48,17 @@
...
-import java.util.concurrent.ExecutorService;
-import java.util.concurrent.ThreadPoolExecutor;
-import java.util.concurrent.TimeUnit;
 
 import static net.bytebuddy.matcher.ElementMatchers.nameContains;
 import static net.bytebuddy.matcher.ElementMatchers.nameStartsWith;
 import static net.bytebuddy.matcher.ElementMatchers.not;
 
+/**
+ * hippo for java Agent
```
Comment (magestacks) [inline agents/threadpool/hippo4j-agent-bootstrap/src/main/java/cn/hippo4j/agent/bootstrap/Hippo4jAgent.java]: It is suggested to change to: `Hippo4j Agent`

File: agents/threadpool/hippo4j-agent-bootstrap/src/main/java/cn/hippo4j/agent/bootstrap/Hippo4jAgent.java (hunk 1 above)
Comment (DDreame) [reply agents/threadpool/hippo4j-agent-bootstrap/src/main/java/cn/hippo4j/agent/bootstrap/Hippo4jAgent.java]: no，beacuse the mehtod 'Permian' has assigned value for it.

File: agents/threadpool/hippo4j-agent-bootstrap/src/main/java/cn/hippo4j/agent/bootstrap/Hippo4jAgent.java (hunk 2 above)
Comment (DDreame) [reply agents/threadpool/hippo4j-agent-bootstrap/src/main/java/cn/hippo4j/agent/bootstrap/Hippo4jAgent.java]: Ok,i will fix it soon.

**Review by magestacks — APPROVED** [review]

**Pull request merged**